plugins/inputs/mongodb/mongodb.go

```diff
@@ -50,10 +55,18 @@ func (*MongoDB) SampleConfig() string {
 }
 
 func (m *MongoDB) Init() error {
-	var tlsConfig *tls.Config
+	if m.DisconnectedServersBehavior == "" {
+		m.DisconnectedServersBehavior = "default"
+	}
+
+	err := choice.Check(m.DisconnectedServersBehavior, DisconnectedServersBehaviors)
+	if err != nil {
+		return err
+	}
```

Comment:
Make the returned error a bit more descriptive
```suggestion
	err := choice.Check(m.DisconnectedServersBehavior, DisconnectedServersBehaviors)
	if err != nil {
		return fmt.Errorf("disconnected_servers_behavior: %w", err)
	}
```